### PR TITLE
Change feature value table to add position field

### PR DESCRIPTION
**Pull request by ks129**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Added position field to feature value table in 9.0.0. Part of PrestaShop/PrestaShop#37042
| Type?             | new feature
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | part of PrestaShop/PrestaShop#35499 and PrestaShop/PrestaShop#32267
| Sponsor company   | -
| How to test?      | After upgrading to 9.0.0, feature value table should have position field.

### CODE REVIEW

**Comment by ps-jarvis** [comment]
Hello @ks129!

This is your first pull request on [autoupgrade](https://github.com/PrestaShop/autoupgrade/) repository of the PrestaShop project.

Thank you, and welcome to this Open Source community!

**Comment by ga-devfront** [comment]
Thank you for your contribution! I have blocked the PR while the PR relating to it is merged on the Core. Do not hesitate to ping me if it is merged and I have not reacted in the meantime.

**Review by Quetzacoalt91 — DISMISSED** [review]
Looks good to me, thank you. Labeling as "Blocked" until the PR is merged on PrestaShop repository.

File: upgrade/php/ps_900_update_configuration.php
Comment (Quetzacoalt91) [inline upgrade/php/ps_900_update_configuration.php]: This method does not seem called by the 9.0.0.sql script.

If you want to add configuration values to the shop, you may refer to this other SQL query:
https://github.com/PrestaShop/autoupgrade/blob/e1658640a47966e6cceaf20a202186886d74fc6f/upgrade/sql/9.0.0.sql#L58-L60

**Review by Quetzacoalt91 — DISMISSED** [review]

**Comment by Quetzacoalt91** [comment]
Hi @ks129,
Can you please rebase this PR so we can proceed with the review? Thanks

**Comment by ks129** [comment]
@Quetzacoalt91 I already rebased it today.

**Comment by Quetzacoalt91** [comment]
Indeed! Sorry for that, I checked the backlog of PRs yesterday and only remembered this morning that I did not post here.

**Review by Quetzacoalt91 — APPROVED** [review]

**Review by AureRita — APPROVED** [review]
Hi @ks129 

Thank you for your PR, I tested it and it seems to works as you can see : 

image


Tested from : 
8.0.4 to 8.2 
8.2 to 9.0.0
8.0.4 to 9.0.0

Because the PR seems to works as expected, It's QA :heavy_check_mark: 

Thank you

**Comment by Quetzacoalt91** [comment]
Thanks @ks129

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
